app/helpers/participants/ideas_helper.rb

```diff
@@ -1,3 +1,13 @@
 module Participants::IdeasHelper
+	def cache_key_for_summary_ideas
+		four_ideas = Idea.limit(4)
```

Comment:
Although retrieve four again from database would not be a big thing, it is always better to reuse the ideas passed by the arguments. You can use `ideas.take(4)` to get the first four ideas. Additionally, it guarantee the consistency between the code in the view and here (e.g. if view sort the ideas by its popularity, and forgot to change the helper function, now the helper function still retrieve the first four rather than first four in terms of popularity)